phylogenetic/defaults/colors.tsv

```diff
@@ -0,0 +1,7 @@
+# Regions
+region	Asia	#447CCD
```

Comment:
_non-blocking_

FYI, Augur now includes a [default color for asia](https://github.com/nextstrain/augur/blob/556aa28e5e1bf175cd2d792569246d99ab023f84/augur/data/colors.tsv?plain=1#L1) so this is only necessary if you want to override the default colors available.

Reply:
I decided to override the default region colors to maintain consistency with other Nextstrain builds. The Asia color is the same as the default colors, but the other regions' colors are not. I am keeping the Asia color in `phylogenetic/defaults/colors.tsv` even though it is the same as the default color, to reduce potential confusion that could arise if Asia were absent from this file.